### ISSUE TITLE
all_options.html: options all glued to following word

### ISSUE THREAD

**Issue by jidanni** [gpsbabel/gpsbabel#1014]
In [all_options.html](https://www.gpsbabel.org/htmldoc-development/all_options.html) the options,
```
    <p><code class="option">-w</code>Work o
    <p><code class="option">-T</code>Enable
    <p><code class="option">-b</code>Proces...
```
will all render glued to the following word. Please add one space in whatever
postprocessor you are using. Thanks.

And
```
-h -?Print help.
```
should not only be unglued:
```
-h -? Print help.
```
but also add a comma:
```
-h, -? Print help.
```

**Comment by robertlipe** [comment]
Hi, Dan.

I agree. That looks bad.

The source,
https://github.com/GPSBabel/gpsbabel/blob/34f9c21aa96f0660d126c428b6f58654556a5148/xmldoc/chapters/use.xml#L443,
has what appears to be significant whitespace but something gets lost on
the way to HTML where the spaces are lost.

<code class="option">-r</code>Work on routes. Thi
I'll delve into Docbook, which has served us well but which doesn't seem to
have a lot of activity around it these days, and figure it out.

OTOH, now that I've fixed the ability to build the HTML doc on my laptop,
it looks like the problem may have taken care of itself. The same source on
my laptop coughs up:

 <code class="option">-r</code> Work on routes.

Which, when rendered to HTML, does contain spaces.

[image: image.png]

So this may "just" be a matter of re-rendering the pages on a system with
the latest (fixed?) Docbook tools as our source didn't change. It looks
fine in 1.6.0 but is malformed like this in 1.7.0, 1.8.0, and -development.

I'm overdue to regenerate and repopulate the entire web site but there have
been a number of interwoven deadlocks that I need to break before I can do
a full push soon. (Amongst other things, I have accessibility issues that
Google's webmaster tools are nagging me about.)

Just so this doesn't get caught in the gravity of the sun, I've rebuilt
that page and copied it up just for you.

https://www.gpsbabel.org/htmldoc-development/all_options.html

**Comment by tsteven4** [comment]
I think this was all fixed in #941 10/30/2022.

@robertlipe , I would be happy to produce the gpsbabel.org target during CI and archive it.  This would alleviate the need to have all the necessary files for generation on ones personal machine.

**Comment by tsteven4** [comment]
@jidanni Is there anything left here?  https://www.gpsbabel.org/htmldoc-development/all_options.html looks good to me.  I think the source was fixed previously as I mentioned and @robertlipe just need to get it to the server.
